ci/jenkins/test.sh

```diff
@@ -531,11 +535,11 @@ function  build_and_deliver_antrea_windows_and_linux_containerd_images {
     ${PRINT_DOCKER_STATUS}
     export_govc_env_var
     # Enable verbose log for troubleshooting.
-    sed -i "s/--v=0/--v=4/g" build/yamls/antrea.yml build/yamls/antrea-windows-containerd.yml
+    sed -i "s/--v=0/--v=4/g" build/yamls/antrea.yml build/yamls/antrea-windows-containerd-with-ovs.yml
```

Comment:
@XinShuYang Is it as plan to run OVS inside container in antrea CI after this change?

Reply:
@wenyingd This change is only for testing , it will be reverted and I have another PR for adding new CI jobs related to containerd-ovs

Reply:
@NamanAg30 If you want a different job, please also update testbed deployment script to add another snapshot for ovs container.